lib/ssl/src/ssl.erl

```diff
@@ -2773,29 +2847,36 @@ eccs(Other) ->
     erlang:error({badarg, Other}).
 
 %%--------------------------------------------------------------------
--doc false.
+-doc(#{title => <<"TLS-1.3 Only Functions">>,
+      since => <<"OTP 22">>}).
+-doc """
+   Returns all supported groups in TLS 1.3
+""".
```

Comment:
```suggestion
    """.
```

If you want to indent the doc string, then you also need to inded the ending quotation mark. There are many places where this is done, I've only commented on this one.